erpnext/stock/doctype/item_variant_settings/item_variant_settings.js

```diff
@@ -4,17 +4,30 @@
 frappe.ui.form.on('Item Variant Settings', {
 	setup: function(frm) {
```

Comment:
Do the options refresh by just being in the `setup` trigger? I have seen instances where options are bugged due to not being refreshed.

Reply:
I've also received random reports of options not getting set. Should we move this to `refresh`?

Reply:
Yes, moving to `refresh` does fix the issue.